cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller.h

```diff
@@ -60,14 +70,15 @@ class CartesianController
 	void showDot(double,double,double,int,double,double,double,std::string);
 	void showLevel(tf::Transform,int,double,double,double,std::string);
 	void timerCallback(const ros::TimerEvent&);
-	void calculateProfile(std::vector<double>*,double,double,double,std::string);
-	void calculateProfileForAngularMovements(std::vector<double> *pathMatrix,double,double,double,double,double,double,double,double,double,std::string,bool);	
-	void generatePath(std::vector<double>*,double,double,double,double,int,std::string);
-	void generatePathWithTe(std::vector<double> *pathArray,double T_IPO, double te, double AcclMax,double Se_max, int steps_max,double start_angle,std::string profile);
+
 	void start_tracking();
 	void stop_tracking();
 	void PoseToRPY(geometry_msgs::Pose pose,double &roll, double &pitch, double &yaw);
```

Comment:
Consistency:
- `functionNames` (not `function_names` or `FunctionNames`)
- `class& name` (not `class &name`)